README.md

```diff
@@ -1,2 +1,14 @@
 # Advanced-CSS3
-In this final project, you’ll get to take everything you’ve learned and use it to build your own simple grid-based framework.
+In this final project, you’ll get to take everything you’ve learned and use it to build your own simple 12 column grid-based framework.
+
+# (- PROJECT: Advanced-CSS3 - )
+I recreated the Microsoft.com website using a custom built framework.
+
+## Authors: 
+### Aaron
+Aaron's Github Profile: https://github.com/Aaron-RN
+
+
+## Github Link
+Github Link: https://github.com/Aaron-RN/Advacned-CSS3
+
```

Comment:
#### `REQUIRE` 
:bulb: Let's add a more descriptive README.
- What are the functions/features of this framework?
- What are technology was used?
- How can one use it?
- Can you add a demonstration?
- Is there an example of a cloned site?
- Since you cloned a site with it can you post a link to the clone (You can use Githack) and the source page?
- What about screenshots: how does the clone look on different screen sizes?

#### The above list are just suggestion, please feel free to make it as descriptive as you want. :ok_hand: